src/flamenco/runtime/fd_executor.c

```diff
@@ -324,11 +324,6 @@ fd_executor_check_replenish_program_cache( fd_exec_txn_ctx_t * txn_ctx ) {
     if( FD_UNLIKELY( err!=FD_ACC_MGR_SUCCESS ) ) {
       continue;
     }
-
-    // https://github.com/anza-xyz/agave/blob/df892c42418047ade3365c1b3ddcf6c45f95d1f1/svm/src/transaction_processor.rs#L254
-    if( ( !fd_account_is_executable( account->const_meta ) ) ) {
-      continue;
-    }
     
```

Comment:
Can we make it clear why this check was removed, and how this new code matches up with Agave.